src/routes/api/users.go

```diff
@@ -8,48 +8,54 @@ import (
 	"onepixel_backend/src/dtos"
 
 	"github.com/gofiber/fiber/v2"
-	"github.com/samber/lo"
 )
 
-var usersController *controllers.UsersController
-
 // UsersRoute /api/v1/users
-func UsersRoute(db *gorm.DB) func(router fiber.Router) {
-	usersController = controllers.NewUsersController(db)
-	return func(router fiber.Router) {
-		router.Post("/", registerUser)
-		router.Post("/login", loginUser)
-		router.Get("/:id", auth.MandatoryAuthMiddleware, getUserInfo)
-		router.Patch("/:id", auth.MandatoryAuthMiddleware, updateUserInfo)
-	}
+func UsersRoute(router fiber.Router, usersController *controllers.UsersController) {
+	router.Get("/", getAllUsers)
+	router.Post("/", func(c *fiber.Ctx) error {
+        return registerUser(c, usersController)
```

Comment:
have you not run go fmt ? the indentation seems off here

Reply:
Yes, hadn't used it and made some alignment errors while resolving conflicts. Used it now.

Reply:
Fixed this along with all test errors.